docs/swagger.json

```diff
@@ -789,6 +799,10 @@
                     "type": "string",
                     "description": "User occupation"
                 },
+                "organization": {
```

Comment:
Field "organization" is repeated thrice, is it necessary?

Reply:
@roopalJazz thanks for the review! this file `swagger.json` is generated automatically by flask-restplus extension. So I just configure within the code and then when I run the flask app, I just export this file. You can use Swagger Editor (https://editor.swagger.io/) to see the output of this file. I think its repeated because of this:

![image](https://user-images.githubusercontent.com/11148726/42771228-61f7b014-891e-11e8-9653-b7c06fc33441.png)

Does it make sense now?

Reply:
@isabelcosta thanks. Also, can you resolve merge conflicts ?

Reply:
@roopalJazz Yes of course!